sunbeam/commands/bootstrap.py

```diff
@@ -60,14 +60,16 @@
 
 
 @click.command()
+@click.option("-a", "--accept-defaults", help="Accept all defaults.", is_flag=True)
+@click.option("-p", "--preseed", help="Preseed file.")
 @click.option(
     "--role",
     default="converged",
     type=click.Choice(["control", "compute", "converged"], case_sensitive=False),
     help="Specify whether the node will be a control node, a "
     "compute node, or a converged node (default)",
 )
-def bootstrap(role: str) -> None:
+def bootstrap(role: str, preseed: str = None, accept_defaults: bool = False) -> None:
```

Comment:
```suggestion
def bootstrap(role: str, preseed: Optional[Path] = None, accept_defaults: bool = False) -> None:
```
`DeployMicrok8sApplicationStep` accepts a path (optional) not a str